04-gas/gas-section-three/gas-section-three.html

```diff
@@ -11,14 +11,15 @@
 <body>
     <section class="gas-section-three-container">
 
-        <section class="gas-section-three-subcontainer">
-
-          <div class="gas-section-three-copy-container">
-            <h2 class="gas-section-three-subtitle">3.</h2>
-            <h1 class="gas-section-three-title">
-              <span class="gas-section-three-number">3.</span>
-              Move your sofa away from the radiator
-            </h1>
+      <section class="gas-section-three-subcontainer">
+  
+        <div class="gas-section-three-copy-container">
+          
+          <div class="gas-section-three-title-container">
+            <h1 class="gas-section-three-title"> <span class="bold">3.Move your sofa </span>away from <span class="bold">radiator</span></h1>
```

Comment:
We need a space between the number and the text like in the Figma
"3. Move your sofa"

Reply:
done